_beacons/win_notify.py

```diff
@@ -171,13 +172,19 @@ def beacon(config):
                     confirm = _add_acl(path, mask, wtype, recurse)
                     sys_check = 1
                 if config[path].get('exclude', False):
-                    _remove_acl(path)
+                    for item in config[path]['exclude']:
+                        if '*' in exclude:
+                            for wildcard_item in glob.iglob(item):
+                                _remove_acl(wildcard_item)
+                        _remove_acl(item)
```

Comment:
> The _remove_acl function is just removing the ACE flag that marks the folder for monitoring. If for some reason someone wants to monitor c:\users and all subfolders, but not c:\users\*\desktop and subfolders, this gives them that option.

I understand that, but `c:\users\*\desktop` isn't a valid path, so I figure you should do `if '*' in exclude, do one thing, else, just remove the acl`. Does that make sense?